### PR TITLE
Drop ocscheme

### PR DESCRIPTION
**Pull request by soltysh**
/assign @juanvallejo 
@deads2k you'll be happy to see this :)

### CODE REVIEW

File: pkg/oc/util/ocscheme/read_internal.go [hunk 1]
```diff
@@ -1,20 +0,0 @@
-package ocscheme
```
Comment (juanvallejo) [inline pkg/oc/util/ocscheme/read_internal.go]: +1

**Comment by soltysh** [comment]
I own the kubectl bits, adding approve manually.

**Pull request merged**